crates/fadroma/ensemble/ensemble.rs

```diff
@@ -384,6 +421,81 @@ impl Context {
                         responses.push(Response::Bank(res));
                     }
                 },
+                CosmosMsg::Staking(msg) => match msg {
+                    StakingMsg::Delegate {
+                        validator,
+                        amount,
+                    } => {
+                        self.bank.writable()
+                            .remove_funds(&sender, vec![amount.clone()])
+                            .expect("Insufficient funds to stake");
```

Comment:
I don't currently have an account for validators because if I added an account then the validator's balance would be query-able, which I don't think is the actual behavior of validator accounts. I see the benefit of using `transfer()` here, but I'm not sure which way better reflects reality.